UKHomeOffice/sgmr-service
# SPCR-383 Implement initial departure form

### AC / Description of changes
Create a departure form using the information the backend currently needs. The form should not have to/from or country inputs (as these are awaiting backend changes). The port input should use the autocomplete component and the file should have validation.

### To Test
- Click on 'Create a new voyage plan'
- You should see a new departure form
- Click on 'Save and continue'
- >All fields should error and error links should scroll to and focus on inputs 
- Enter an invalid date (32/13/2022) and invalid time (24:00)
- >Errors should read 'You must enter a valid departure date/time'
- Enter a date in the past
- >Error should read 'You must enter a departure date in the future'
- Enter current date but a time in the past
- >Error should read 'You must enter a departure time in the future'. 
- Enter valid details
- Click save and **exit**
- >You should be taken to the dashboard and see the voyage in the draft
- Enter valid details
- Click save and **continue**
- >You should be taken to the V1 arrival form

### Notes

Tests will be added later

---
* remember to merge to feature branches not master *
